runtime/kusama/src/lib.rs

```diff
@@ -308,6 +308,7 @@ parameter_types! {
 	pub const MaxNominatorRewardedPerValidator: u32 = 64;
 	// quarter of the last session will be for election.
 	pub const ElectionLookahead: BlockNumber = EPOCH_DURATION_IN_BLOCKS / 4;
+	pub const MaxIterations: u32 = 5;
```

Comment:
Why only 5 here?

Reply:
And a better name `StakingMaxIterations` or whatever would also be useful.

Reply:
For Kusama and Polkadot I set to 5 to be more cautious not to stall solution submission for too long. If all is well we can increase it in a further upgrade.

Reply:
Okay